tests/sandbox/test_create_referral.py

```diff
@@ -0,0 +1,60 @@
+from typing import Callable, Dict, Iterable
+
+import pytest
+import asserts
+
+from SandboxTest import SandboxTest
+from data import Actor
+from utils import HttpMethod
+from requests import Response
+
+
+@pytest.mark.sandbox
+class TestCreateReferral(SandboxTest):
+    @pytest.fixture
+    def unauthorised_actors(self) -> Iterable[Actor]:
+        return [Actor.SPC, Actor.SPCA]
+
+    @pytest.fixture
+    def call_endpoint(
+        self,
+        send_rest_request: Callable[[HttpMethod, str, Actor], Response],
+        load_json: Callable[[str], Dict[str, str]],
+    ) -> Callable[[Actor], Response]:
+        return lambda actor, headers={}: send_rest_request(
+            HttpMethod.POST,
+            "FHIR/STU3/ReferralRequest/$ers.createReferral",
+            actor,
+            json=self._load_json_data(actor, load_json),
+            headers=headers,
+        )
+
+    @pytest.fixture
+    def allowed_business_functions(self) -> Iterable[str]:
+        return ["REFERRING_CLINICIAN", "REFERRING_CLINICIAN_ADMIN"]
+
+    @pytest.mark.parametrize("actor", [Actor.RC, Actor.RCA])
+    def test_success(
+        self,
+        call_endpoint: Callable[[Actor], Response],
+        load_json: Callable[[str], Dict[str, str]],
```

Comment:
load_json defined in conftest.py as Callable[[str], str]

Reply:
Updated this also 🙂